internal/tiltfile/os/os.go

```diff
@@ -77,18 +85,70 @@ func osName() string {
 	return "posix"
 }
 
-func environ() (starlark.Value, error) {
-	env := os.Environ()
-	result := starlark.NewDict(len(env))
-	for _, e := range os.Environ() {
-		pair := strings.SplitN(e, "=", 2)
-		err := result.SetKey(starlark.String(pair[0]), starlark.String(pair[1]))
-		if err != nil {
-			return nil, err
-		}
+func getenv(t *starlark.Thread, fn *starlark.Builtin, args starlark.Tuple, kwargs []starlark.Tuple) (starlark.Value, error) {
+	var varname starlark.Value
+	var defaultVal starlark.Value = starlark.None
+	err := starkit.UnpackArgs(t, fn.Name(), args, kwargs,
+		"varname", &varname,
+		"value?", &defaultVal,
+	)
+	if err != nil {
+		return nil, err
+	}
+
+	varnameStr, ok := value.AsString(varname)
+	if !ok {
+		return nil, fmt.Errorf("varname must be a string, actual: %s", varname)
```

Comment:
Does defaultVal also need to be enforced to be a string?

Reply:
surprisingly no!

```
python -c "import os; print(os.getenv('x', {}))"
{}
```